doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -94,8 +94,15 @@ For OpenStack control nodes we use the BIOS parameters for the basic profile def
 | …|  
  
 -	How many nodes to meet SLA
+    - Minimum 2 nodes for high availibility using VRRP.
 -	HW specifications
+    - 3 NICs card are needed if we want to isolate the different flows :
+         - 1 NIC for Tenant Network
+         - 1 NIC for External Network
+         - 1 NIC for Other Networks (PXE, Mngt ...)
 -	Sizing rules
+    - Scale out of network node is not easy 
+    - DVR can be an option for large deployment (see more detail
```

Comment:
see more detail where?

Reply:
I complete it with (see more detail on chapter 4.3.1.5) which will be completed with DVR description with the next PR of @pgoyal01

Reply:
okay. thanks.